### ISSUE TITLE
为什么交叉熵计算时目标标签都是0

### ISSUE THREAD

**Issue by AlonZhao** [MCZhi/DTPP#30]
zhiyu 你好，    
label = torch.zeros(scores.shape[0], dtype=torch.long).to(scores.device) # why zeros？
    irl_loss = F.cross_entropy(scores, label)
在计算损失函数时，交叉熵计算为什么设定label 标签都为零呢？

**Comment by SteveHao74** [comment]
同问这个问题

**Comment by HaoquanLiu** [comment]
Same question here.

**Comment by Jerome-Cong** [comment]
#5 I refer to the explanation here
